src/applications/personalization/components/NotInMPIError.unit.spec.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { mount } from 'enzyme';
```

Comment:
This test file already existing for an older version of the NotInMPIError component that I deleted.